Demo/TableViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+class TableCell: UITableViewCell {
+
+}
+
+class TableViewController: SweetTableViewController {
+    lazy var data: [String] = {
```

Comment:
`let data = ["Some", "Stringing", "Going", "On"]`

I think you could use this instead.